### DISCUSSION TITLE
I have to unistall my antivirus. why? i don't want to

### DISCUSSION THREAD

**Discussion by MacrosZero** [Atlas-OS/Atlas#1165] [Help & Support] [answered]
I have to unistall my antivirus. why? i don't want to

**Comment by Ast3risk-ops** [comment]
you don't? The reinstall process wipes it anyway and you just need to turn off defender during playbook application. Afterward, you can reinstall your normal AV.

**Comment by he3als** [comment, marked as answer]
You haven't followed the installation guide to reinstall Windows. A reinstall of Windows is necessary for Atlas.

https://docs.atlasos.net/getting-started/installation/
